src/main/java/models/Player.java

```diff
@@ -0,0 +1,37 @@
+package models;
+
+
+public class Player
+{
+    public int score;
+    public int money;
+    public int initBetAmount;
+    public int isBet;
+
+    public Player()
+    {
+        score = 0;
+        money = 100;
+        isBet = 0; //amount player bets
+        initBetAmount = 5;
```

Comment:
I think this needs to be changed to 2. In the instructions the initial bet is 2 I think.